src/mono/wasm/debugger/DebuggerTestSuite/Support.cs

```diff
@@ -68,7 +71,31 @@ async Task OnMessage(string method, JObject args, CancellationToken token)
                     NotifyOf(READY, args);
                     break;
                 case "Runtime.consoleAPICalled":
-                    Console.WriteLine("CWL: {0}", args?["args"]?[0]?["value"]);
+                    String type = args?["type"]?.Value<string>();
+                    List<string> consoleArgs = new List<string>();
+                    foreach (var arg in args?["args"])
+                    {
+                        consoleArgs.Add(arg?["value"].ToString());
+                    }
+
+                    int position = 1;
+                    string first = consoleArgs[0];
+                    string output = _consoleArgsRegex.Replace(first, (_) => $"{consoleArgs[position++]}");
+                    if (position == 1)
+                    {
+                        // first arg wasn't a format string so concat things together
+                        // with a space instead.
+                        StringBuilder builder = new StringBuilder(first);
+                        for (position = 1; position < consoleArgs.Count(); position++)
+                        {
+                            builder.Append(" ");
+                            builder.Append(consoleArgs[position]);
+                        }
+                        builder.Append("\n");
+                        output = builder.ToString();
```

Comment:
Maybe `output = string.Join(' ', consoleArgs) + "\n";`?

Reply:
I'd rather do it this way because I may go back to a format string if I do the logging changes I was considering.